lib/orbital/ext/active_merchant/active_merchant.rb

```diff
@@ -59,6 +59,54 @@ def commit(order, message_type, trace_number=nil)
                          :cvv_result => OrbitalGateway::CVVResult.new(response[:cvv2_resp_code])
                      })
       end
+
+      # A – Authorization request
+      def authorize(money, creditcard, options = {})
+        order = build_new_order_xml(AUTH_ONLY, money, options) do |xml|
+          add_creditcard(xml, creditcard, options[:currency], options[:cvv_indicator_visa_discover])
+          add_address(xml, creditcard, options)
+          if @options[:customer_profiles]
+            add_customer_data(xml, creditcard, options)
+            add_managed_billing(xml, options)
+          end
+        end
+        commit(order, :authorize, options[:trace_number])
+      end
+
+      # AC – Authorization and Capture
+      def purchase(money, creditcard, options = {})
+        order = build_new_order_xml(AUTH_AND_CAPTURE, money, options) do |xml|
+          add_creditcard(xml, creditcard, options[:currency], options[:cvv_indicator_visa_discover])
+          add_address(xml, creditcard, options)
+          if @options[:customer_profiles]
+            add_customer_data(xml, creditcard, options)
+            add_managed_billing(xml, options)
+          end
+        end
+        commit(order, :purchase, options[:trace_number])
+      end
+
+      def add_creditcard(xml, creditcard, currency=nil, cvv_indicator_visa_discover=false)
```

Comment:
Nit: when overriding, I usually just end up passing the full `options` as the last parameter (in case further options are needed one day).